bridges/modules/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -341,8 +341,8 @@ impl<T: Config<I>, I: 'static> SendXcm for Pallet<T, I> {
 		// versioned message to the sibling bridge hub. However, the local bridge hub may have a
 		// higher XCM version than the remote `dest`. Once again, it is better to discard such
 		// messages here than at the bridge hub (e.g., to avoid losing funds).
-		let destination_version = T::DestinationVersion::get_version_for(dest_ref)
-			.ok_or(SendError::DestinationUnsupported)?;
+		let destination_version =
+			T::DestinationVersion::get_version_for(dest_ref).ok_or(SendError::NotApplicable)?;
```

Comment:
As required by 
https://github.com/Snowfork/polkadot-sdk/blob/67b7d649c74fba140b1e7dcf818b8d2cc2860cc3/polkadot/xcm/src/v4/traits.rs#L265,  the impl of `SendXcm` should only raise error `NotApplicable` to give chance to the [next impl of the Tuple](https://github.com/Snowfork/polkadot-sdk/blob/67b7d649c74fba140b1e7dcf818b8d2cc2860cc3/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs#L665) to try.

Reply:
Good catch!